build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.3.2"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.4.1-beta"
```

Comment:
You may want to wait till this is outta beta, otherwise you'll be changing it again very shortly